Dockerfile

```diff
@@ -1,23 +1,16 @@
 # Build the manager binary
 FROM registry.access.redhat.com/ubi9/go-toolset:9.5-1739801907 as builder
 
+USER 1001
 # Copy the Go Modules manifests
-COPY go.mod go.mod
-COPY go.sum go.sum
+COPY --chown=1001:0 go.mod go.mod
+COPY --chown=1001:0 go.sum go.sum
 # cache deps before building and copying source so that we don't need to re-download as much
 # and so that source changes don't invalidate our downloaded layer
 RUN go mod download
 
 # Copy the go source
-COPY main.go main.go
-COPY api/ api/
-COPY cache/ cache/
-COPY controllers/ controllers/
-COPY loader/ loader/
-COPY metadata/ metadata/
-COPY metrics/ metrics/
-COPY syncer/ syncer/
-COPY tekton/ tekton/
+COPY --chown=1001:0 . .
```

Comment:
With this change, we are now copying things into the image we don't really need like `hack/` and `integration-tests/pipelines`

Reply:
I  checked the other services and it's the same.  I guess it because sealight analyzes code coverage. @flacatus , can you help to answer it?

Reply:
can you use .dockerignore to not copy them

Reply:
Why? Why not just keep the multiple COPY directives like we have but add the `--chown` to each?

Reply:
@johnbieren when Sealights do go instrumentation they create additional packages to your code and we need to copy it to the container in order to build

Reply:
Ah, got it. The .dockerignore thing sounds good to me then

Reply:
Do we need `config\` ,  go.mod and go.sum included in .dockerignore file? And I saw only `release-in-cluster-local` Degraded during konflux deployment in the PR's test. Not sure what may cause it?

Reply:
I mean anything that is not currently in a COPY directive but exists in the root of the repo I'd think should be added to the ignore

Reply:
But the network_policy.yaml file is under config/manager dir.  If `config/` is in .dockerignore, how the change of network_policy.yaml affect the operator deployment?